src/app/util/Util.js

```diff
@@ -57,10 +57,28 @@ export const checkFeatureFlagActivated = (featureFlagList, componentStateObject)
   });
 };
 
+// Given a link, return the link with 'http' and strip https if present
+export const formatUrl = (link) => {
```

Comment:
This isn't necessary at all but I think this could be simplified using a regex. But I tend to abuse regexes so it is entirely up to you
```
if (link.match(/^http[s]?:\/\//)) { return `http://${link}` }
return link
```

Reply:
Regex totally appeals to the "brevity and elegance" part of my brain, but I struggle with writing and reading it, so I'm gonna leave it